src/inmanta/protocol/endpoints.py

```diff
@@ -132,10 +132,11 @@ class SessionEndpoint(Endpoint, CallTarget):
         An endpoint for clients that make calls to a server and that receive calls back from the server using long-poll
     """
 
+    _client: "SessionClient"
```

Comment:
Shouldn't this be optional?

Reply:
no, because that makes all other code more complex. It is set in start so, should be available.